pages/common/nix.md

```diff
@@ -0,0 +1,23 @@
+# nix
+
+> Utilities for the Nix language and store.
+
+- Search for a package via its name or description:
+
+`nix search {{search_term}}`
+
+- Start a nix shell with the specified packages available:
```

Comment:
N should be capital here to be consistent with the rest of the page.

Reply:
Nice catch!

Reply:
@aaronduino this doesn't seem to have been changed yet — marking as unresolved so this doesn't get lost in the conversation.

Reply:
Looking at the most recent file view (clicking on files changed, not on this page), it seems to be fixed, no? https://github.com/tldr-pages/tldr/pull/2808/files

Reply:
@waldyrious, is this ok to merge?

Reply:
Yep, this is indeed fixed.